CenterForOpenScience/osf.io
# [ENG-2132] Track transition metrics

## Purpose

We want to track all moderation transitions via our ES database.

## Changes

- adds transitions metric class that runs on all transitions and has method for relent query
- adds new view for running a metrics query
- add tests

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify all transitions from `/metrics/registries_moderation/transitions/` endpoint is accurate.

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation

Internal feature, no new docs

## Side Effects

Travis is persnickety when it comes to ES tests.

## Ticket

https://openscience.atlassian.net/browse/ENG-2132
